src/starlark.rs

```diff
@@ -1,102 +1,43 @@
-use std::fs;
+use starpls::StarPls;
```

Comment:
```suggestion
use starpls::Starpls;
```

starpls doesn't capitalize the p